src/main/java/ValidationResults.java

```diff
@@ -0,0 +1,39 @@
+import lombok.*;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+
+public class ValidationResults {
+
+    public int total;
+    public int valid;
+    public int zeroPriceCounter;
+    public int startDateErrorsCounter;
+    public int ticketTypeErrorsCounter;
+    public int ticketClassErrorsCounter;
+
+    public void setTotal() {
+        this.total += 1;
```

Comment:
+= 1 Could be replaced with just ++.  Also naming of the methods could be ambiguous,  they not actually set value but increment it instead